src/Orckestra.Composer.C1.Content.EN-CA/Package/Datas/Composite.Data.Types.IPagePlaceholderContents_administrated.xml

```diff
@@ -43,7 +43,7 @@
   <Add PageId="f3dbd28d-365f-4d3e-91c3-7b730b39b294" PlaceHolderId="pageHeader" Content="&lt;html xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&#xA;&#x9;&lt;head&gt;&#xA;&#x9;&#x9;&lt;f:function xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot; name=&quot;Composer.Header.PageHeader&quot; /&gt;&#xA;&#x9;&lt;/head&gt;&#xA;&#x9;&lt;body&gt;&#xA;&#x9;&lt;/body&gt;&#xA;&lt;/html&gt;" ChangeDate="2016-02-10 12:28:05Z" ChangedBy="admin" PublicationStatus="published" SourceCultureName="en-CA" />
-  <Add PageId="f3dbd28d-365f-4d3e-91c3-7b730b39b294" PlaceHolderId="body" Content="&lt;html xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&#xA;&#x9;&lt;head&gt;&#xA;&#x9;&lt;/head&gt;&#xA;&#x9;&lt;body&gt;&#xA;&#x9;&#x9;&lt;div class=&quot;jumbotron&quot;&gt;&#xA;&#x9;&#x9;&#x9;&lt;f:function name=&quot;Composer.PageTitle&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot; /&gt;&#xA;&#x9;&#x9;&#x9;&lt;p class=&quot;lead&quot;&gt;The Reference Application is a e-commerce Starter Site developed by Orckestra A/S to create new projects efficiently and quickly.&lt;/p&gt;&#xA;&#x9;&#x9;&#x9;&lt;p&gt;This is the C1 CMS version.&lt;/p&gt;&#xA;&#x9;&#x9;&#x9;&lt;hr class=&quot;my-4&quot; /&gt;&#xA;&#x9;&#x9;&#x9;&lt;a class=&quot;btn btn-primary btn-lg&quot; href=&quot;#&quot; role=&quot;button&quot;&gt;Learn more&lt;/a&gt;&#xA;&#x9;&#x9;&lt;/div&gt;&#xA;&#x9;&#x9;&lt;h1 class=&quot;text-center mb-2&quot;&gt;This week's specials&lt;/h1&gt;&#xA;&#x9;&#x9;&lt;f:function name=&quot;Composer.Queries.ProductSet.Top&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot;&gt;&#xA;&#x9;&#x9;&#x9;&lt;f:param name=&quot;QueryName&quot; value=&quot;FeaturedProducts&quot; /&gt;&#xA;&#x9;&#x9;&#x9;&lt;f:param name=&quot;Number&quot; value=&quot;4&quot; /&gt;&#xA;&#x9;&#x9;&lt;/f:function&gt;&#xA;&#x9;&lt;/body&gt;&#xA;&lt;/html&gt;" ChangeDate="2022-08-26 11:35:26Z" ChangedBy="admin" CreationDate="2020-04-28 18:38:50Z" CreatedBy="admin" PublicationStatus="published" SourceCultureName="en-CA" />
```

Comment:
I deployed and home page contents are empty
Try remove the VersionId="94578974-decf-4222-8185-5b03d81cbc39"

Reply:
commited the fix